lib/functions/rootfs/rootfs-create.sh

```diff
@@ -235,7 +235,11 @@ function create_new_rootfs_cache_via_debootstrap() {
 
 	# Remove `machine-id` (https://www.freedesktop.org/software/systemd/man/machine-id.html)
 	# Note: This will mark machine `firstboot`
```

Comment:
And this is not true anymore. As you cited from man:
`If /etc/machine-id exists and is empty, a boot is not considered the first boot`

Reply:
Thanks, forgot to remove that comment. Done now.